site/docs/v1/tech/identity-providers/index.adoc

```diff
@@ -76,4 +95,143 @@ An email address or domain may be provided in the `login_hint` request parameter
+| Link On Username | No | Username | Has the user's username and users that do not exist in the identity provider can have access.
+| Link On Username. Do not create the user if they do not exist | Yes | Username | Has the user's username and you don't want users that do not exist in the identity provider to have access.
+|===
+
+
+Some identity providers don't provide a username and/or email.  In those instances, it is recommended to consider using a pending link or creating an anonymous link.  Both of these options enable you to link the user without an email/username in the response from the IdP.
+
+== Linking and Create Registration
+
+The [field]#Linking strategy# and [field]#Create registration# configurations are related to each other, but distinct. 
+The [field]#Linking strategy# controls how a User is created in FusionAuth based on information returned from the remote identity provider. [field]#Create registration# controls if the User created in FusionAuth is registered for a given Application.
+
+== Linking Examples
+
+Here are some walkthroughs of linking scenarios. A user logs in to a remote Identity Provider when the FusionAuth hosted login pages are being used. Similar behavior is available via the link:/docs/v1/tech/apis/identity-providers/[Identity Provider API]. 
+
+It doesn't matter if the Identity Provider is a social provider like Facebook or an enterprise provider like an OIDC or SAML compatible identity server. 
+
+=== Link By Email For Existing User
+
+Richard is logging into NewApp. He doesn't have an account in FusionAuth.
+
+* Richard clicks on the 'Login With Hooli' button on the login screen. 
+* He logs in with the `richard@piedpiper.com` account. 
+* He is redirected to FusionAuth but sees an error. No access is allowed.
+
+Here's an example of the error page:
+
+image::identity-providers/idp-linking-user-must-exist.png[Error when a user must exist for successful linking.,width=1200,role=bottom-cropped]
+
+==== Link By Email For Existing User With an Existing Account
+
+Richard is logging into NewApp. He has an account in FusionAuth.
+
+* Richard clicks on the 'Login With Hooli' button on the login screen. 
+* He logs in with the `richard@piedpiper.com` account. 
+* He is redirected to FusionAuth and logs in successfully. Access is allowed.
+* The FusionAuth account with the email `richard@piedpiper.com` is linked to the Hooli `richard@piedpiper.com` account.
+
+=== Link On Email 
+
+Richard is logging into NewApp. He doesn't have an account in FusionAuth.
+
+* Richard clicks on the 'Login With Hooli' button on the login screen. 
+* He logs in with the `richard@piedpiper.com` account. 
+* He is redirected to FusionAuth.
+* A new account is created in FusionAuth with the email `richard@piedpiper.com`. 
+* The new FusionAuth account with the email `richard@piedpiper.com` is linked to the Hooli `richard@piedpiper.com` account.
+
+=== Link By Username For Existing User
```

Comment:
I am not sure if this reads correctly.  

Link By Username For Existing User --> do not create a user if they don't exist